migrations/config/gemfiles/common/Gemfile

```diff
@@ -6,13 +6,17 @@ source "https://rubygems.org"
 ruby ">= 3.2.2"
 
 # `activesupport` gem needs to be in sync with the Rails version of Discourse, see `/Gemfile`
-gem "activesupport", "< 7.1", require: "active_support"
+lock_file = %w[Gemfile.lock ../Gemfile.lock].detect { File.exist?(_1) }
+activesupport_version =
```

Comment:
Here we parse the core Gemfile.lock directly. This allows us to get the `activesupport` version really in use, that way it’s always in sync.

Reply:
Fantastic change! Thanks!